plugins/by-name/ltex-extra/default.nix

```diff
@@ -2,16 +2,24 @@
   lib,
   helpers,
   config,
+  options,
   ...
 }:
 with lib;
+let
+  lspCfg = config.plugins.lsp;
```

Comment:
If you like, we could also have:

```nix
  lspOpts = options.plugins.lsp;
```

If you think that makes anything more readable?

No preference here.